src/components/Answer.js

```diff
@@ -1,15 +1,19 @@
 import React, {useState}from "react";
+import Image from "../assets/Image"
 
 
-const Answer = (answerObject) => {
+const Answer = (data) => {
     const [selected, setSelected] = useState(false)
-    if (answerObject.content.img) {
+    const handleClick = () => {
+        setSelected(!selected)
+    }
+    if (data.content.img) {
         return (
-            <img
-                onClick={() => setSelected(!selected)}
+            <Image
+                handleClick={handleClick}
                 className={`answer-img ${selected ? 'answer-img--active' : ''}`}
-                src={process.env.PUBLIC_URL + answerObject.content.img.src}
-                alt={answerObject.content.img.alt}
+                src={process.env.PUBLIC_URL + data.content.img.src}
```

Comment:
@rachellebosman What did we decide on using the public map?

Reply:
I thought we decided it was oke to put in the public map. Fastest solution, so we could continue on working on the project and not waste too much time.